eureka-core/src/main/java/com/netflix/eureka/registry/RemoteRegionRegistry.java

```diff
@@ -145,20 +140,22 @@ public RemoteRegionRegistry(EurekaServerConfig serverConfig,
         } catch (UnknownHostException e) {
             logger.warn("Cannot find localhost ip", e);
         }
-        EurekaServerIdentity identity = new EurekaServerIdentity(ip);*/
-        // FIXME 2.0
-        // discoveryApacheClient.addFilter(new EurekaIdentityHeaderFilter(identity));
+        EurekaServerIdentity identity = new EurekaServerIdentity(ip);
```

Comment:
Should these lines still be here? There's no more `FIXME 2.0` so I'm not sure.

Reply:
yeah, those are for the non-experimental client I still need to add back. If you look at the top of the comment block, there is a `FIXME: 2.0` still